packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextViewManager.kt

```diff
@@ -0,0 +1,219 @@
+      val update = extraData as ReactTextUpdate
+      val spannable: Spannable = update.text
+      @Suppress("DEPRECATION")
+      if (update.containsImages()) {
+        TextInlineImageSpan.possiblyUpdateInlineImageSpans(spannable, view)
+      }
+      view.setText(update)
+
+      // If this text view contains any clickable spans, set a view tag and reset the accessibility
+      // delegate so that these can be picked up by the accessibility system.
+      val accessibilityLinks: ReactTextViewAccessibilityDelegate.AccessibilityLinks = ReactTextViewAccessibilityDelegate.AccessibilityLinks(spannable)
+      view.setTag(
+        R.id.accessibility_links,
+        if (accessibilityLinks.size() > 0) accessibilityLinks else null
+      )
+      ReactTextViewAccessibilityDelegate.resetDelegate(
+        view, view.isFocusable(), view.getImportantForAccessibility()
+      )
+    }
+  }
+
+  override fun createShadowNodeInstance(): ReactTextShadowNode = ReactTextShadowNode(reactTextViewManagerCallback)
+
+  public fun createShadowNodeInstance(
+    reactTextViewManagerCallback: ReactTextViewManagerCallback?
+  ): ReactTextShadowNode = ReactTextShadowNode(reactTextViewManagerCallback)
+
+  override fun getShadowNodeClass(): Class<ReactTextShadowNode> = ReactTextShadowNode::class.java
+
+  override fun onAfterUpdateTransaction(view: ReactTextView) {
+    super.onAfterUpdateTransaction(view)
+    view.updateView()
+  }
+
+  override fun needsCustomLayoutForChildren(): Boolean = true
+
+  override fun updateState(
+    view: ReactTextView, props: ReactStylesDiffMap, stateWrapper: StateWrapper
+  ): Any? {
+    SystraceSection("ReactTextViewManager.updateState").use { s ->
+      val stateMapBuffer = stateWrapper.stateDataMapBuffer
+      return if (stateMapBuffer != null) {
+        getReactTextUpdate(view, props, stateMapBuffer)
+      } else {
+         null
+      }
+    }
+  }
+
+  private fun getReactTextUpdate(
+    view: ReactTextView,
+    props: ReactStylesDiffMap,
+    state: MapBuffer
+  ): Any {
+    val attributedString: MapBuffer = state.getMapBuffer(TX_STATE_KEY_ATTRIBUTED_STRING.toInt())
+    val paragraphAttributes: MapBuffer = state.getMapBuffer(TX_STATE_KEY_PARAGRAPH_ATTRIBUTES.toInt())
+    val spanned: Spannable =
+      TextLayoutManager.getOrCreateSpannableForText(
+        view.getContext(), attributedString, reactTextViewManagerCallback
+      )
+    view.setSpanned(spanned)
+
+    try {
+      val minimumFontSize: Float =
+          paragraphAttributes.getDouble(TextLayoutManager.PA_KEY_MINIMUM_FONT_SIZE.toInt())
+              .toFloat()
+      view.setMinimumFontSize(minimumFontSize)
+    } catch (e: IllegalArgumentException) {
+      // T190482857: We see rare crash with MapBuffer without PA_KEY_MINIMUM_FONT_SIZE entry
+      FLog.e(
+        TAG,
+        "Paragraph Attributes: %s",
+        // if (paragraphAttributes != null) paragraphAttributes.toString() else "<empty>"
```

Comment:
Needs fix
```suggestion
        "Paragraph Attributes: %s",
```

Reply:
This crash actually went away at some point, so we could also remove the try/catch entirely